tools/functional-tester/etcd-tester/stresser.go

```diff
@@ -28,8 +29,10 @@ func init() { grpclog.SetLogger(plog) }
 type Stresser interface {
 	// Stress starts to stress the etcd cluster
 	Stress() error
-	// Cancel cancels the stress test on the etcd cluster
-	Cancel()
+	// Puase pauses the stresser on stressing the etcd cluster
```

Comment:
`Pause stops the stresser from sending requests to etcd. Resume by calling Stress.`